apps/google-analytics-4/frontend/src/components/main-app/ChartContent.tsx

```diff
@@ -26,8 +26,8 @@ const ChartContent = (props: Props) => {
       <LineChart
         dataValues={parseRowViews()}
         xAxisLabels={parseRowDates()}
-        tooltipMetricLabel={''}
-        accessibilityLabel={'Analytics line chart'}
+        tooltipMetricLabel="Page views"
```

Comment:
nit: when you run `prettier --write .` is this what happens, I thought we we're single quoting

Reply:
good catch here - I am not sure how it got thru, will reconfirm my prettier/eslint configuration. Thanks for noticing!